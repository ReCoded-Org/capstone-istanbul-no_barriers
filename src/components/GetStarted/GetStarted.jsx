import React from "react";

const GetStarted = () => {

}

export default GetStarted