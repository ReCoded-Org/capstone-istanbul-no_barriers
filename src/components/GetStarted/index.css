:root {
  --grey-bg-color: #eee;
}

.signUp {
  background-color: orange;
  border-color: var(--grey-bg-color);
}

.contactUs {
  background-color: var(--grey-bg-color);
  color: orange;
  border-color: orange;
}

.getStarted {
  background-color: var(--grey-bg-color);
  padding: 2rem;
  text-align: center;
}
