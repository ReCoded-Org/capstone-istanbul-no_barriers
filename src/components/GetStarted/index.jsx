import React from "react";
import { useHistory } from "react-router-dom";
import { useTranslation } from "react-i18next";
import Container from "react-bootstrap/Container";
import Row from "react-bootstrap/Row";
import Col from "react-bootstrap/Col";
import Button from "react-bootstrap/Button";
import "./index.css";

const GetStarted = () => {
  const { t } = useTranslation();
  const history = useHistory();

  const handleSignUp = () => history.push("/signup");

  return (
    <Container fluid className="getStarted">
      <Row>
        <Col md="6">
          <h6>{t("homePage.getStarted.ready")}</h6>
          <p>{t("homePage.getStarted.parag")}</p>
        </Col>
        <Col xs md="1.5">
          <Button
            className="signUp"
            variant="primary"
            size="sm"
            onClick={handleSignUp}
          >
            {t("homePage.getStarted.signUp")}
          </Button>
        </Col>
        <Col xs md="1.5">
          <Button className="contactUs" variant="primary" size="sm">
            {t("homePage.getStarted.contactUs")}
          </Button>
        </Col>
        <Col xs md="3" />
      </Row>
    </Container>
  );
};

export default GetStarted;
