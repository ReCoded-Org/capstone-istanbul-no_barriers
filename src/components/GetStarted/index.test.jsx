import React from "react";
import renderer from "react-test-renderer";
import GetStarted from "./index.jsx";

it("renders GetStarted correctly", () => {
  const tree = renderer.create(<GetStarted />).toJSON();
  expect(tree).toMatchSnapshot();
});
