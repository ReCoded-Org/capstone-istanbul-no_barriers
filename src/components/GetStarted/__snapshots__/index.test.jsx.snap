// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders GetStarted correctly 1`] = `
<div
  className="getStarted container-fluid"
>
  <div
    className="row"
  >
    <div
      className="col-md-6"
    >
      <h6>
        homePage.getStarted.ready
      </h6>
      <p>
        homePage.getStarted.parag
      </p>
    </div>
    <div
      className="col-md-1.5 col"
    >
      <button
        className="signUp btn btn-primary btn-sm"
        disabled={false}
        onClick={[Function]}
        type="button"
      >
        homePage.getStarted.signUp
      </button>
    </div>
    <div
      className="col-md-1.5 col"
    >
      <button
        className="contactUs btn btn-primary btn-sm"
        disabled={false}
        type="button"
      >
        homePage.getStarted.contactUs
      </button>
    </div>
    <div
      className="col-md-3 col"
    />
  </div>
</div>
`;
